### DISCUSSION TITLE
Text Field prevent losing focus when tapping anywhere on the chat

### DISCUSSION THREAD

**Discussion by Davenzo** [flyerhq/flutter_chat_ui#661] [Ideas & Requests]
When tapping outside the TextField it loses focus and the keyboard is dismissed. It would be good if this was settable as I want users to be able to copy a message text or select it without the keyboard closing like it works on Telegram.

**Comment by Davenzo** [comment]
Can I get a feedback about this? Thanks

**Comment by demchenkoalex** [comment]
I don't add new features to `v1` so I keep this opened to implement in `v2`, no timeline unfortunately. If this is pressing I suggest a fork for now.

**Comment by syacko** [comment]
I have a question about v2 and controlling the chat input focus. I'm using dev five, and I don't see focusNode supported. Am I missing something? I have limited experience with Flutter, so please point me in the right direction.

**Comment by syacko** [comment]
I have opened an issue (bug) report. Also, I have working code that I can check-in or provide to you. Let me know.

**Reply by demchenkoalex** [reply]
replied in the issue
